### ISSUE TITLE
dependency issue

### ISSUE THREAD

**Issue by yinfangchen** [ucsb-seclab/pretender#4]
May I ask where the `./scripts/install_dependencies.sh` is ?

**Comment by mariusmue** [comment]
Hi @DarknessChen!

We checked back on our side - we used install_dependencies before properly packaging Pretender.
All it did was:
```
#!/usr/bin/env bash
pip install scipy numpy
pip install statsmodels
```

However, everything is now handled by the setup.py - hence, the README was just outdated, which is now fixed with 4ac769beb5bd1a19806fafc808569e9a1841a4af.

Thanks for reporting this issue!

**Comment by yinfangchen** [comment]
Thanks @mariusmue! Really appreciate your works.
